module/purpose/Soccer/src/Soccer.cpp

```diff
@@ -68,14 +68,14 @@ namespace module::purpose {
             if (self_penalisation.context == GameEvents::Context::SELF) {
                 emit(std::make_unique<ResetWebotsServos>());
                 emit<Task>(std::unique_ptr<FindPurpose>(nullptr));
-                emit<Task>(std::make_unique<StandStill>());
+                // emit<Task>(std::make_unique<StandStill>());
             }
         });
 
         on<Trigger<Unpenalisation>>().then([this](const Unpenalisation& self_unpenalisation) {
             // If the robot is unpenalised, stop standing still and find its purpose
             if (self_unpenalisation.context == GameEvents::Context::SELF) {
-                emit<Task>(std::unique_ptr<StandStill>(nullptr));
+                // emit<Task>(std::unique_ptr<StandStill>(nullptr));
```

Comment:
Commented out?? Is this for future development?

Reply:
see second dot point in the main PR comment